static/bidder-info/adocean.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://{{.Host}}"
+endpoint: "https://{{.Host}}.adocean.pl"
```

Comment:
@mmuras thanks for making this change. As you mentioned in PR description, as of now it is not possible to use fully static domain at bidder side.

But should consider that usage of partial dynamic urls is being discouraged and prebid team in future will be working to avoid usage of dynamic subdomain in endpoint url.

Major concerns with such usage are,

- security concerns
The security aspect is alleviated by using a fixed top level domain. Due to the potential harm to hosts, we are strict in this requirement. We are working towards fixing the few adapters currently in violation.

- connection performance
The connection performance advice is for your benefit. Client specific subdomains prevent Prebid Server from reusing connections across your clients which results in more connections needed for your adapter. The issue gets worse the more successful you are in attracting new clients. Hosts may choose to disable your adapter due to this behavior.

Reply:
Thank you for pointing this out, but We can't change it to single domain for now.

Reply:
There has been talk about further narrowing down the restriction here. We might at some point in the future reduce this to a specific whitelist of allowed domain names, rather than unlimited subdomain. If we go this route, it is likely that the list of allowed domains can be passed as a config parameter in the host config rather than being hard coded into the adapter. That would be a level of restriction between the current any subdomain allowed and only a singular endpoint allowed.

We would also be interested in hearing arguments as to why multiple domains may be necessary for an adapter, as it currently seems to be a carryover from Prebid.JS, where the browser would be exposed to the endpoint URLs.

Reply:
Multiple domains are required because our infrastructure currently does not have a single entry to connect to. Each client can have (and bigger ones have) own infrastructure with custom settings, and the only thing in common is top level domain.